infoblox_client/utils.py

```diff
@@ -79,7 +79,7 @@ def safe_json_load(data):
     try:
         return jsonutils.loads(data)
     except ValueError:
-        LOG.warn("Could not decode reply into json: %s", data)
+        LOG.warning("Could not decode reply into json: %s", data)
```

Comment:
is this available for both python and oslo_log packages?

Reply:
Yes, I checked the usage docs and they use warning as well. The warn method has been deprecated for a long time now :)